src/schemas/patchSchemas/taxonomyPatchSchemas.ts

```diff
@@ -0,0 +1,81 @@
+import { z } from "zod";
+import { referenceObjectSchema } from "../referenceObjectSchema.js";
+
+type TaxonomyTermValue = {
+  name: string;
+  codename?: string;
+  external_id?: string;
+  terms?: TaxonomyTermValue[];
+};
+
+const taxonomyTermValueSchema: z.ZodType<TaxonomyTermValue> = z.object({
+  name: z.string(),
+  codename: z.string().optional(),
+  external_id: z.string().optional(),
+  terms: z.lazy(() => z.array(taxonomyTermValueSchema)).optional(),
+});
+
+const addIntoOperationSchema = z.object({
+  op: z.literal("addInto"),
+  reference: referenceObjectSchema
+    .optional()
+    .describe(
+      "Parent term reference. Omit to add at root level of taxonomy group.",
+    ),
+  value: taxonomyTermValueSchema.describe("Term data to add"),
+  before: referenceObjectSchema
+    .optional()
+    .describe("Position before this term. Mutually exclusive with 'after'."),
+  after: referenceObjectSchema
+    .optional()
+    .describe("Position after this term. Mutually exclusive with 'before'."),
+});
+
+const moveOperationSchema = z.object({
```

Comment:
I don't like those "hydra/multi" operations. Try to split it into multiple operations. That way we can get rid of the confusions. Apply elsewhere as well.